api/v1/flow/flow.proto

```diff
@@ -663,9 +663,11 @@ message ServiceUpsertNotification {
     ServiceUpsertNotificationAddr frontend_address = 2;
     repeated ServiceUpsertNotificationAddr backend_addresses = 3;
     string type = 4;
-    string traffic_policy = 5;
+    string traffic_policy = 5 [deprecated = true];
```

Comment:
Do we have an idea when we plan to remove this? I realize this could break Hubble flow handling so we may want to wait a few releases. But it would be nice to add a comment like `// TODO: Remove after v1.16` or similar.

Reply:
FWIW we usually do not plan to remove deprecated fields for compatibility reasons (e.g. `reply` or `drop_reason` in the Flow message).